recipe/bld.bat

```diff
@@ -23,5 +24,5 @@ nmake install
 if errorlevel 1 exit 1
 
 :: Move everything 1-level down.
-xcopy %LIBRARY_INC%\freetype2\freetype\*.* %LIBRARY_INC%\freetype2\ /s /e || exit 1
-rmdir %LIBRARY_INC%\freetype2\freetype /s /q
+move %LIBRARY_INC%\freetype2\freetype %LIBRARY_INC% || exit 1
+move %LIBRARY_INC%\freetype2\ft2build.h %LIBRARY_INC% || exit 1
```

Comment:
P.S. I'm just raising the question, not intending to block anything here. I'm completely OK with doing it, just want to make sure we are not being wagged by the tail (context for those who don't have English as their first language: the expression "tail wagging the dog")